pay-api/src/pay_api/services/base_payment_system.py

```diff
@@ -308,7 +312,8 @@ def _refund_and_create_credit_memo(invoice: InvoiceModel, refund_partial: List[R
         payment_account.flush()
 
         try:
-            PaymentSystemService._send_credit_notification(payment_account, refund_amount)
+            if send_credit_notification:
+                PaymentSystemService._send_credit_notification(payment_account, refund_amount)
```

Comment:
EFT doesn't send a credit notification currently. It goes to the EFT unsettled amount that clients cannot see. They would see a short name refund notification which is a separate flow.